lldb/source/Utility/Status.cpp

```diff
@@ -268,7 +243,7 @@ ErrorType Status::GetType() const {
       result = eErrorTypeMachKernel;
     else if (error.isA<Win32Error>())
       result = eErrorTypeWin32;
-    else if (error.isA<ExpressionError>())
+    else if (error.isA<ExpressionErrorBase>())
```

Comment:
I'm wondering if this would look nicer if we added a `GetLLDBErrorType` function to  our `ClonableError` class.